src/app/lista-nominal/page.tsx

```
@@ -0,0 +1,156 @@
+import type { getListDataProps } from "@services/lista-nominal/ListaNominal";
+import type { ExtendedPanelSelectorWithCardsProps } from './ListaNominal';
+import { getCardsData } from "@/services/lista-nominal/cards";
+import { CardProps } from '@impulsogov/design-system/dist/molecules/Card/Card';
+import { baseURL } from '@/utils/baseURL';
+import { CardDetailsMap, getCardsProps } from "@/helpers/cardsList";
+import { captureException } from "@sentry/nextjs";
+
+const ListaNominalPage = async() => {
+    // Dados mockados que virão do CMS. Quantidade e conteúdo varia com a lista.
+    const cardsDetails: CardDetailsMap = {
+        "COM_CONSULTA_AFERICAO_PRESSAO": {
```

Biome (1.9.4)

[error] This object property name part should be in camelCase.



(lint/style/useNamingConvention)